docs/tech_primer.md

```diff
@@ -0,0 +1,157 @@
+# Smart-Ready Building Technical Primer
+
+This document provides  primer for smart-ready building assembly, to essentially
+answer the question *"What does Smart Ready mean?"* and *"How do we know it to be
+true?"* Specifically, it addresses the delta between a traditional site and
+smart-ready sites: places where new bleeding edge building technology is being
+used.
+
+# Key Terms & Definitions
+
+## Building Models
+
+There are different levels of "digital maturity" for different buildings. Each
+one has a slightly different scope in terms of what it means to be "complete."
+For the most part, this document addresses the scope of a Smart Ready building:
+-   **Legacy**: It works, but does not meet requirements or guidelines.
+-   **Compliant**: Meets basic security and networking requirements.
+-   **Smart Ready**: On-prem integrated with the cloud using UDMI & DBO.
+-   **Digital Building**: Completely integrated with back-end services.
+
+## Device
+
+Hardware that exists in the building that can change state in a way that can be machine communicated. A potentiometer or a flow switch is a device if it is monitored by a digital system. A commissioning valve or damper with no communication is not a device. 
+
+## Devices Classification
+
+Various adjectives qualify the different flavors of a "device" found in a building. More than one can apply, and each has a specific technical meaning/definition (i.e. they are not subjective).
+
+### Connectivity
+
+How data is moved around on-prem
+
+-   **Passive**: The device has only passive components (thermistor, dry contact)
```

Comment:
I don't understand what "passive" means in this context -- can you give a specific example for this?  It feels like it might be getting out-of-scope for what needs to be differentiated in this context... The comparison would of course be "active" -- but I'm still not sure why that would matter here.

Reply:
I think the intent was that electrically speaking some building "devices" **are** just passive components (e.g. a thermistor), but in this context I think we can remove it as it's _analogues_ with analogue (they're both considered analogue inputs/outputs and utilise 0-10V/4-20mA)